src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/FormLoginFactory.php

```diff
@@ -29,7 +29,7 @@ public function __construct()
         $this->addOption('username_parameter', '_username');
         $this->addOption('password_parameter', '_password');
         $this->addOption('csrf_parameter', '_csrf_token');
-        $this->addOption('intention', 'authenticate');
+        $this->addOption('csrf_token_id', 'authenticate');
```

Comment:
2.8 is missing a deprecation+bc-layer for renaming intention to csrf_token_id. Could someone give it a try? @stof? Any other @symfony/deciders?